task/sast-unicode-check/0.1/README.md

```diff
@@ -0,0 +1,30 @@
+# sast-unicode-check task
+
+## Description:
+
+The sast-unicode-check task uses [find-unicode-control](https://github.com/siddhesh/find-unicode-control.git) tool to perform Static Application Security Testing (SAST) to look for non-printable unicode characters in all text files in a source tree.
+
+## Parameters:
+
+| name                         | description                                                                                                                                   | Default Value                                                                                   | Required |
+|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|----------|
+| FIND_UNICODE_CONTROL_GIT_URL | URL from repository to find unicode control.                                                                                                  | "https://github.com/siddhesh/find-unicode-control.git#c2accbfbba7553a8bc1ebd97089ae08ad8347e58" | Yes      |
+| FIND_UNICODE_CONTROL_ARGS    | arguments for find-unicode-control command.                                                                                                   | "-p bidi -v -d -t"                                                                              | Yes      |
```

Comment:
I think that `Required` = `Yes` is meaningless for parameters with a default value.  In case the parameter is not provided, the default value will be used.  My understanding of `Required` is that the task will fail to start if the parameter is not provided.

Reply:
Changed `FIND_UNICODE_CONTROL_GIT_URL` and `FIND_UNICODE_CONTROL_ARGS` to `No`